### PR TITLE
feat(clerk-js): Add descriptors and translations for account switcher actions

### PR DESCRIPTION
**Pull request by octoper**
## Description

This PR adds some descriptors and translation keys for the actions inside the account switcher and fixes some borders issues in the account switcher to mach the designs.

**Descriptors added:**
- `accountSwitcherActionButton`
- `accountSwitcherActionButtonIconBox`
- `accountSwitcherActionButtonIcon`

**Translation keys added:**
- `signIn.accountSwitcher.action__addAccount`
- `signIn.accountSwitcher.action__signOutAll`

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Comment by octoper** [comment]
!preview

**Comment by clerk-cookie** [comment]
Hey @octoper, your preview is currently deploying and should be available shortly.

| Status | Preview | Updated (UTC) |
| :-- | :-- | :-- |
| :cook: _Deploying..._ | [Inspect](https://javascript-preview-h81a961rl.clerkstage.dev) | Feb 12, 2024 02:42 PM |

**Review by panteliselef — APPROVED** [review]

File: packages/clerk-js/src/ui/components/OrganizationList/shared.tsx [hunk 1]
```diff
@@ -8,15 +8,22 @@ import { SwitchArrowRight } from '../../icons';
 import type { ThemableCssProp } from '../../styledSystem';
 import { common } from '../../styledSystem';
 
-export const PreviewListItems = (props: PropsWithChildren) => {
+export const PreviewListItems = (
+  props: PropsWithChildren<{
+    sx?: ThemableCssProp;
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/OrganizationList/shared.tsx:13]: You are not using this anywhere 🤔

File: packages/clerk-js/src/ui/components/UserButton/SessionActions.tsx [hunk 2]
```diff
@@ -165,23 +185,25 @@ export const SignOutAllActions = (props: SignOutAllActionsProps) => {
...
-          borderRadius: t.radii.$lg,
-        })}
+        sx={[
+          t => ({
+            backgroundColor: t.colors.$transparent,
+            padding: `${t.space.$2} ${t.space.$3}`,
+            borderBottom: 'none',
+            borderRadius: t.radii.$lg,
+          }),
+        ]}
```
Comment (anagstef) [inline packages/clerk-js/src/ui/components/UserButton/SessionActions.tsx:206]: Maybe you wanted to pass the `sx` here?

File: packages/clerk-js/src/ui/elements/Card/CardAction.tsx [hunk 3]
```diff
@@ -2,23 +2,27 @@ import type { CardActionId } from '@clerk/types';
 import React from 'react';
 
 import { descriptors, Flex, Text } from '../../customizables';
-import type { PropsOfComponent } from '../../styledSystem';
+import type { PropsOfComponent, ThemableCssProp } from '../../styledSystem';
 import { RouterLink } from '../RouterLink';
 
 type CardActionProps = Omit<PropsOfComponent<typeof Flex>, 'elementId'> & {
   elementId?: CardActionId;
+  sx?: ThemableCssProp;
```
Comment (anagstef) [inline packages/clerk-js/src/ui/elements/Card/CardAction.tsx:10]: Isn't `sx` already included in `PropsOfComponent<typeof Flex>` ?

File: packages/clerk-js/src/ui/components/OrganizationList/shared.tsx (hunk 1 above)
Comment (octoper) [reply packages/clerk-js/src/ui/components/OrganizationList/shared.tsx:13]: Yeah this is a leftover from an attempt not needed, updating the code!

File: packages/clerk-js/src/ui/elements/Card/CardAction.tsx (hunk 3 above)
Comment (octoper) [reply packages/clerk-js/src/ui/elements/Card/CardAction.tsx]: Good catch, removed!

**Comment by octoper** [comment]
!preview

**Comment by octoper** [comment]
!preview

**Review by anagstef — APPROVED** [review]
💯

**Pull request merged**